atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -401,7 +404,7 @@ def add_log_type(output):
         repositories = [image.to_str() for image in output_images
                         if image.tag != 'latest']
         arch = os.uname()[4]
```

Comment:
I hope brew is fine with this and we won't have to do some translation (in future it would be for the best if brew provided the arch)